osu.Game.Rulesets.Mania/UI/DrawableManiaRuleset.cs

```diff
@@ -48,6 +50,10 @@ public class DrawableManiaRuleset : DrawableScrollingRuleset<ManiaHitObject>
         protected new ManiaRulesetConfigManager Config => (ManiaRulesetConfigManager)base.Config;
 
         private readonly Bindable<ManiaScrollingDirection> configDirection = new Bindable<ManiaScrollingDirection>();
+        private readonly Bindable<double> configTimeRange = new BindableDouble();
+
+        // Stores the current speed adjustment active in gameplay.
+        private readonly Track speedAdjustmentTrack = new TrackVirtual(1000);
```

Comment:
is there a specific reason for the 1000 duration here? i think i'd feel better with 0 otherwise